tests/components/AddonList.test.js

```diff
@@ -40,21 +40,21 @@ describe('<AddonList />', () => {
     const openPage = () => {};
     const openModal = () => {};
     const addonList = mount( <AddonList appList={appList} openPage={openPage}  openModal={openModal}/ > );
-    
```

Comment:
Consider : I observe that this PR contains a lot of format and spacing changes that are not quite relevant to  the task. Daniel recommended that it is wise to separate formatting changes from the changes that show what directly applies to  the feature you were working on. This way,  the PR becomes less noisy.

Reply:
My editor's linter automatically removes some linting issues like white space